one-page.js

```diff
@@ -112,11 +133,17 @@ function inlineAssets(projectPath) {
                         } break;
 
                         case "json": {
-                            if (asset.type !== 'model' && asset.type !== 'animation') {
+                            // The model and animation loader assumes that the base64 URL will be loaded as a binary
+                            if ((asset.type !== 'model' && asset.type !== 'animation')) {
                                 mimeprefix = "data:application/json";
                             }
                         } break;
 
+                        case "css":
+                        case "html": {
+                            mimeprefix = "data:text/plain";
+                        } break;
```

Comment:
The curly braces around all your case statements are unnecessary.

Reply:
Ah, old habit of C++ code style to always use braces. Would you prefer them to be removed for consistency with other repos?

Reply:
I would. But up to you. 😉